crates/sui-storage/src/write_ahead_log.rs

```diff
@@ -247,19 +322,20 @@ where
 }
 
 #[async_trait]
-impl<'a, C: 'a> WriteAheadLog<'a, C> for DBWriteAheadLog<C>
+impl<'a, C: 'a, ExecutionOutput: 'a> WriteAheadLog<'a, C> for DBWriteAheadLog<C, ExecutionOutput>
```

Comment:
@mystenmark not sure if the lifetime parameter for `ExecutionOutput` is correct here. Added because the compiler told me to, but don't understand that construct well enough to determine if it's safe or not

Reply:
This is fine - I think its required because of the signature of `begin_tx` and `read_one_recoverable_tx` - its saying that anything you pass in for C and ExecutionOutput will have to be valid as long as the self parameter for those functions.